internal/rpcserver/rpcserver.go

```diff
@@ -4692,8 +4671,8 @@ func handleTicketFeeInfo(_ context.Context, s *Server, cmd interface{}) (interfa
 				feeInfo, err := ticketFeeInfoForRange(s, i-winLen, i,
 					stake.TxTypeSStx)
 				if err != nil {
-					return nil, rpcInternalError(err.Error(),
-						"Could not obtain ticket fee info")
+					const context = "Could not obtain ticket fee info"
+					return nil, rpcInternalErr(err, context)
```

Comment:
I think we can use a one-liner? It's not too long. I noticed we also do this in a few other places.

Reply:
It probably depends on your tab stop size.  dcrd preferentially uses 4 spaces for tab stop.  You can find some older code that used 8 before it was being enforced to 4, but that would just make it even wider.

![image](https://github.com/decred/dcrd/assets/2115102/aa88618b-5302-461b-ad3d-6229f050098a)

Reply:
> It probably depends on your tab stop size

I use 4. Anyway, it's just a nit. Some "contexts" (containing about 4 - 5 words) are used without declaring a const value, but I think it's neater for longer "contexts".

Reply:
Right.  If it fits, I didn't bother declaring a separate const for it.  When it doesn't fit, I went with the constant since I find it easier to read than splitting up a string parameter across lines.